src/github.com/getlantern/fronted/masquerade.go

```diff
@@ -138,7 +138,7 @@ func (vms *verifiedMasqueradeSet) doVerify(masquerade *Masquerade) bool {
 	go func() {
 		start := time.Now()
 		httpClient := vms.dialer.HttpClientUsing(masquerade)
-		req, err := http.NewRequest("HEAD", "http://www.google.com/humans.txt", nil)
+		req, err := http.NewRequest("HEAD", "http://geo.getiantem.org/lookup", nil)
```

Comment:
Check out my changes in https://github.com/getlantern/lantern/pull/3073 @fffw

It turns out it's not the URL that triggers the 403 but the enproxy headers!